### PR TITLE
Feature/86bz4fzma

### PR DESCRIPTION
**Pull request by NatalySheludko**
add config for windows, create global styles for project

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -10,22 +10,28 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "axios": "^1.6.8",
+    "@emotion/react": "11.11.4",
+    "@emotion/styled": "11.11.5",
+    "@mui/icons-material": "5.16.0",
+    "@mui/material": "5.16.0",
+    "@mui/styled-engine-sc": "6.0.0-alpha.18",
+    "@vitejs/plugin-react": "^4.3.1",
```
Comment (anastasia1383) [inline ui/package.json:18]: lock versions

File: ui/src/main.tsx [hunk 2]
```diff
@@ -1,10 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import { App } from './App.tsx';
-import './index.css';
+import React from "react";
```
Comment (anastasia1383) [inline ui/src/main.tsx:1]: organize imports pls
so libs on top, styles are the last

File: ui/src/main.tsx [hunk 3]
```diff
@@ -1,10 +1,17 @@
...
+import ReactDOM from "react-dom/client";
+import { App } from "./App.tsx";
+import "./index.css";
+import GlobalStyle from "./styles/GlobalStyle.ts";
+import { ThemeProvider } from "styled-components";
+import theme from "./styles/theme.ts";
```
Comment (anastasia1383) [inline ui/src/main.tsx:7]:
```suggestion
import React from "react";
import ReactDOM from "react-dom/client";
import { ThemeProvider } from "styled-components";
import { App } from "./App.tsx";
import GlobalStyle from "./styles/GlobalStyle.ts";
import theme from "./styles/theme.ts";
import "./index.css";
```

**Review by anastasia1383 — CHANGES_REQUESTED** [review]
Good job! left some commentts

**Review by ptrodion — DISMISSED** [review]

**Review by anastasia1383 — APPROVED** [review]
Good job!

File: ui/src/main.tsx (hunk 2 above)
Comment (NatalySheludko) [reply ui/src/main.tsx:1]: Imports organize

File: ui/src/main.tsx (hunk 3 above)
Comment (NatalySheludko) [reply ui/src/main.tsx]: Imports organize

**Review by VelMakarova — APPROVED** [review]

**Pull request merged**